crates/bevy_image/src/image.rs

```diff
@@ -850,16 +849,14 @@ impl Image {
     }
 
     /// Resizes the image to the new size, by removing information or appending 0 to the `data`.
-    /// Does not properly resize the contents of the image, but only its internal `data` buffer.
+    /// Does not properly scale the contents of the image.
     pub fn resize(&mut self, size: Extent3d) {
         self.texture_descriptor.size = size;
```

Comment:
It's probably out of scope for this PR, but setting the size when we don't have any data feels weird. I'd put this in the condition 🤔